docs-source/content/faq/namespace-management.md

```diff
@@ -47,7 +47,11 @@ suspendOnDebugStartup: false
 
```

Comment:
General comment: This file has references to 'helm upgrade' and 'helm get' that weren't updated.  I thought those needed different treatment for helm 3 and helm 2?

Reply:
Helm upgrade and get is compatible for what we do. Note this md assume default namespace for all the helm part.